Changelog.md

```diff
@@ -2,6 +2,119 @@
+    * [Pull request #391 from gazebosim/chapulina/5_to_6](https://github.com/gazebosim/gz-physics/pull/391 from gazebosim/chapulina/5_to_6)
+    * [Pull request #Merge branch 'main' into chapulina/5_to_6](https://github.com/gazebosim/gz-physics/pull/Merge branch 'main' into chapulina/5_to_6)
+
+1. dartsim: fix handling inertia matrix pose rotation
+    * [Pull request #351](https://github.com/gazebosim/gz-physics/pull/351)
+
+1. ABI tweaks to help bullet-featherstone
+    * [Pull request #395](https://github.com/gazebosim/gz-physics/pull/395)
+
+1. Add code coverage ignore file
+    * [Pull request #388](https://github.com/gazebosim/gz-physics/pull/388)
+
+1. Fix bullet classic entity management
+    * [Pull request #393](https://github.com/gazebosim/gz-physics/pull/393)
+
+1. Ignition to Gazebo transition
+    * [Pull request #386](https://github.com/gazebosim/gz-physics/pull/386)
+    * [Pull request #Fixed merge - ignition to gz](https://github.com/gazebosim/gz-physics/pull/Fixed merge - ignition to gz)
+    * [Pull request #378](https://github.com/gazebosim/gz-physics/pull/378)
+    * [Pull request #368](https://github.com/gazebosim/gz-physics/pull/368)
+    * [Pull request #356](https://github.com/gazebosim/gz-physics/pull/356)
+    * [Pull request #355](https://github.com/gazebosim/gz-physics/pull/355)
+    * [Pull request #358](https://github.com/gazebosim/gz-physics/pull/358)
+    * [Pull request #353](https://github.com/gazebosim/gz-physics/pull/353)
+    * [Pull request #348](https://github.com/gazebosim/gz-physics/pull/348)
+    * [Pull request #346](https://github.com/gazebosim/gz-physics/pull/346)
+    * [Pull request #390](https://github.com/gazebosim/gz-physics/pull/390)
+
+1. Partial source and cmake migrations
+    * [Pull request #381](https://github.com/gazebosim/gz-physics/pull/381)
+
+1. Move png file
+    * [Pull request #381](https://github.com/gazebosim/gz-physics/pull/381)
+
+1. Restore CXX_STANDARD export from core library
+    * [Pull request #377](https://github.com/gazebosim/gz-physics/pull/377)
+
+1. Fixed warning SDFfeatures_TEST - dartsim
+    * [Pull request #370](https://github.com/gazebosim/gz-physics/pull/370)
+
+1. Update GoogleTest to latest version
+    * [Pull request #369](https://github.com/gazebosim/gz-physics/pull/369)
+
+1. Use new Joint APIs for Parent/Child name
+    * [Pull request #361](https://github.com/gazebosim/gz-physics/pull/361)
+
+1. Bullet: Added collision shape to Capsule and Ellipsoid
+    * [Pull request #350](https://github.com/gazebosim/gz-physics/pull/350)
+
+1. Version bumps and removal of deprecations
+    * [Pull request #344](https://github.com/gazebosim/gz-physics/pull/344)
+    * [Pull request #341](https://github.com/gazebosim/gz-physics/pull/341)
+    * [Pull request #Bumps in garden: depend on ign-math7, ign-common5 and sdformat13](https://github.com/gazebosim/gz-physics/pull/Bumps in garden: depend on ign-math7, ign-common5 and sdformat13)
```

Comment:
The automated changelog generator fails in some cases as in this one. Can you find the correct PR number for this?

Reply:
Removed as it was a part of version bumps